aat/src/aat/resources/features/F-077/S-331.td.json

```diff
@@ -0,0 +1,230 @@
+{
+  "_guid_": "S-331",
+  "_extends_": "F-077_Test_Base_Data",
+  "title": "Should return all valid base types",
+  "specs": [
+    "an active profile in CCD",
+    "has the 200 OK code"
+  ],
+  "expectedResponse": {
+    "responseCode" : 200,
+    "responseMessage" : "OK",
+    "headers": {
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "Content-Encoding" : "gzip",
+      "Content-Type" : "application/json;charset=UTF-8",
+      "Date" : "[[ANYTHING_PRESENT]]",
+      "Content-Length" : "[[ANYTHING_PRESENT]]",
+      "Expires" : "0",
+      "Pragma" : "no-cache",
+      "Vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "X-Frame-Options" : "DENY",
+      "X-Xss-Protection" : "1; mode=block"
```

Comment:
Most or all lines here could be inherited from a common object, like Common_200_Response.